fvm/Cargo.toml

```diff
@@ -31,7 +31,7 @@ lazy_static = "1.4.0"
 derive-getters = "0.2.0"
 derive_more = "0.99.17"
 replace_with = "0.1.7"
-filecoin-proofs-api = { version = "13", default-features = false }
+filecoin-proofs-api = { version = "14", default-features = false }
```

Comment:
Is this network breaking? I.e., will updating to version 14 cause a hard-fork?

Reply:
The answer _should_ be no, but I will let @cryptonemo confirm.

Reply:
The answer is *yes*, this is `nv19` scope and beyond.

Reply:
To clarify, it's yes once the new PoSt Proof types are used/accepted.  If you continue using the old PoSt proof types in the new version of proofs, it's compatible.